notes/09-integration-tests.md

```diff
@@ -71,47 +77,47 @@ let button = find('input[type="submit"]');
 let select = find('select');
 ```
 
-If you wanted to add some special autocomplete awesomeness, you could give your editor a clue (via [JSDoc comments](http://usejsdoc.org/tags-type.html)) of the types we expect `find` to return in these cases
+If you wanted to add some special autocomplete awesomeness, you could give your editor a clue (via [JSDoc comments](http://usejsdoc.org/tags-type.html)) of the types we expect `find` to return in these cases.
 
 ```js
 let button = /** @type {HTMLInputElement} */ (find('input[type="submit"]'));
 let select = /** @type {HTMLSelectElement} */ (find('select'));
 ```
 
-We will also need to import `find` from `@ember/test-helpers` by updating line 3: 
+We will also need to import `find` from `@ember/test-helpers` by updating line 3:
 
 ```js
 import { render, find } from '@ember/test-helpers';
 ```
 
-Now we can write two assertions against these
+Now we can write two assertions against these:
 
 ```js
 assert.equal(select.value, '', 'Initially, no user is selected');
 assert.equal(button.disabled, true, 'Initially the button is disabled');
 ```
 
-Go back to http://localhost:4200/tests?filter=login-form&nolint and you should see the nice assertion labels showing up
+Go back to http://localhost:4200/tests?filter=login-form&nolint and you should see the nice assertion labels showing up:
 
 ![assertion-labels](./img/09-integration-tests/assertion-labels.png)
 
-Create a new test immediately below the first one, and name it something like
+Create a new test immediately below the first one, and name it something like:
 
 ```ts
-test('after selecting a user "Sign In" button enabled', async function(assert) {
+test('after selecting a user "Sign In" button enabled', async function (assert) {
   // ...
 });
 ```
 
-define the test body as follows
+Define the test body as follows:
 
 ```ts
 // Render the component
 await render(hbs`<LoginForm />`);
 
 // Pluck off the DOM elements we care about
-let button = /** @type {HTMLInputElement} */ (find('input[type="submit"]'));
-let select = /** @type {HTMLSelectElement} */ (find('select'));
+let button = /** @type {HTMLInputElement} */ find('input[type="submit"]');
+let select = /** @type {HTMLSelectElement} */ find('select');
```

Comment:
roll this change back please, in the interest of making sure vscode autocomplete works, with the `@type` JSDoc annotation